packages/http-client-csharp/generator/Microsoft.Generator.CSharp.Input/src/InputTypes/InputApiKeyAuth.cs

```diff
@@ -11,7 +11,7 @@ public InputApiKeyAuth(string name = "", string? prefix = null) : base()
             Prefix = prefix;
         }
 
-        public string Name { get; }
-        public string? Prefix { get; }
+        public string Name { get; internal set; }
```

Comment:
I don't know that we want internal setters on everything.  Ony on the things we know for sure we have use cases to edit after deserialization from codemodel.json

Reply:
well fair point.
I will remove those unnecessary setters.